CMakeLists.txt

```diff
@@ -17,6 +17,12 @@
 #
 
 cmake_policy(SET CMP0048 NEW)  # project_VERSION* variables populated from project(... VERSION x.x.x) string
+
+# Fix behavior of CMAKE_CXX_STANDARD when targeting macOS. See https://gitlab.kitware.com/cmake/cmake/issues/15943
+if (POLICY CMP0025)
+  cmake_policy(SET CMP0025 NEW)
+endif ()
```

Comment:
I think this problem may come about because `project` and `cmake_required_version` are in the wrong order. You usually want the latter first [psi](https://github.com/psi4/psi4/blob/master/CMakeLists.txt#L2-L6) because it sets the policies based on the min version. `CMP0025` was in cmake 3.0, so it should already be `NEW`.